CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 #Changelog
 
+## 1.15.2
+* Bug - Fixed a bug where container state information wasn't reported. [#1067](https://github.com/aws/amazon-ecs-agent/pull/1067)
```

Comment:
Can you fix the number, it should be 1076.

Reply:
done